app/views/result_filters/funding/new.html.erb

```diff
@@ -15,7 +15,7 @@
         Salaried courses are not eligible for bursaries, scholarships or student finance. There are usually no course fees to pay.
       </p>
       <%= form_with url: funding_path do |form| %>
-        <%= render 'result_filters/hidden_fields', exclude_keys: ["funding"], form: form %>
+        <%= render "shared/hidden_fields", exclude_keys: ["funding"], form: form %>
```

Comment:
This is only shared amongst the `result_filter` views so I'd leave it in that folder. I can live it with it moving though if you disagree 👍

Reply:
I moved it because I need it on the results page too

Reply:
Ok... fair enough. Should we name the file something different then do you think? "shared/result_querystring_hidden_fields" or something like that. Just thinking 'hidden_fields' doesn't really give you much to go on about what it is.

Reply:
Fair enough. What do you think about renaming the file itself then? 'hidden_fields' doesn't give a lot of description of what the file is. What about `results_querystring_hidden_fields` or something like that?